src/app/entry/superDescriptorLanguage.ts

```diff
@@ -0,0 +1,110 @@
+}
+
+const superCWL: SuperDescriptorLanguage = {
+  value: 'CWL',
+  friendlyName: 'Workflow Descriptor ',
+  descriptorPathPattern: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(cwl|yaml|yml)',
+  descriptorPathPlaceholder: 'e.g. /Dockstore.cwl',
+  toolDescriptorEnum: ToolDescriptor.TypeEnum.CWL,
+  workflowDescriptorEnum: Workflow.DescriptorTypeEnum.CWL,
+  plainTRS: 'PLAIN-CWL',
+  sourceFileTypeEnum: [SourceFile.TypeEnum.DOCKSTORECWL],
+  toolTab: {
+    rowIdentifier: 'tool\xa0ID',
+    secondColumnHeading: 'Tool Excerpt'
+  }
+};
+
+const superWDL: SuperDescriptorLanguage = {
+  value: 'WDL',
+  friendlyName: 'Workflow Description Language',
+  descriptorPathPattern: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.wdl$',
+  descriptorPathPlaceholder: 'e.g. /Dockstore.wdl',
+  toolDescriptorEnum: ToolDescriptor.TypeEnum.WDL,
+  workflowDescriptorEnum: Workflow.DescriptorTypeEnum.WDL,
+  plainTRS: 'PLAIN-WDL',
+  sourceFileTypeEnum: [SourceFile.TypeEnum.DOCKSTOREWDL],
+  toolTab: {
+    rowIdentifier: 'task\xa0ID',
+    secondColumnHeading: 'Task Excerpt'
+  }
+};
+
+const superNFL: SuperDescriptorLanguage = {
+  value: 'NFL',
+  friendlyName: 'Nextflow',
+  descriptorPathPattern: '^^/([^/?:*|<>]+/)*[^/?:*|<>]+.(config)',
+  descriptorPathPlaceholder: 'e.g. /nextflow.config',
+  toolDescriptorEnum: ToolDescriptor.TypeEnum.NFL,
+  workflowDescriptorEnum: Workflow.DescriptorTypeEnum.NFL,
+  plainTRS: 'PLAIN-NFL',
+  sourceFileTypeEnum: [SourceFile.TypeEnum.NEXTFLOW, SourceFile.TypeEnum.NEXTFLOWCONFIG],
+  toolTab: {
+    rowIdentifier: 'process\xa0name',
+    secondColumnHeading: 'Process Excerpt'
+  }
+};
+
+const superService: SuperDescriptorLanguage = {
+  value: 'service',
+  friendlyName: 'generic placeholder for services',
+  // This is not really applicable
+  descriptorPathPattern: '.*',
+  descriptorPathPlaceholder: 'e.g. /.dockstore.yml',
+  toolDescriptorEnum: ToolDescriptor.TypeEnum.SERVICE,
+  workflowDescriptorEnum: Workflow.DescriptorTypeEnum.Service,
+  plainTRS: 'PLAIN-SERVICE',
+  sourceFileTypeEnum: [],
+  toolTab: {
+    rowIdentifier: 'tool\xa0ID',
+    secondColumnHeading: 'Service'
+  }
+};
+
+const superGalaxy: SuperDescriptorLanguage = {
+  value: 'gxformat2',
+  friendlyName: 'Galaxy Workflow Format 2',
+  descriptorPathPattern: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(ga|yaml|yml)',
+  descriptorPathPlaceholder: '/Dockstore.ga',
+  toolDescriptorEnum: ToolDescriptor.TypeEnum.GXFORMAT2,
+  workflowDescriptorEnum: null,
+  plainTRS: '<FILL-IN>',
+  sourceFileTypeEnum: [SourceFile.TypeEnum.DOCKSTOREGXFORMAT2],
+  toolTab: {
+    rowIdentifier: 'tool\xa0ID',
+    secondColumnHeading: 'Tool Excerpt'
+  }
+};
+
+export const superUnknown: SuperDescriptorLanguage = {
+  value: null,
+  friendlyName: null,
+  descriptorPathPattern: '.*',
+  descriptorPathPlaceholder: '',
+  toolDescriptorEnum: null,
+  workflowDescriptorEnum: null,
+  plainTRS: null,
+  sourceFileTypeEnum: [],
+  toolTab: {
+    rowIdentifier: 'tool\xa0ID',
+    secondColumnHeading: 'Tool Excerpt'
```

Comment:
secondColumnHeading variable name needs to be more descriptive, alternatively comments somewhere

Reply:
suggestion?